src/test/java/org/glassfish/el/test/OperatorTest.java

```diff
@@ -101,7 +103,19 @@ public void testMisc() {
         ValueExpression v = elm.getExpressionFactory().createValueExpression(
                 elm.getELContext(), "#${1+1}", Object.class);
         Object ret = v.getValue(elm.getELContext());
-        assertEquals(ret, "#2");
+        assertEquals("#2", ret);
+
+        v = elm.getExpressionFactory().createValueExpression(
+                elm.getELContext(), "##{1+1}", Object.class);
+        assertEquals("#2", v.getValue(elm.getELContext()));
```

Comment:
@TomasHofman If you want you can also use the `evaluateExpression` method for this. But it's OK, it's not important.

Reply:
Right -- done.